components/experimental/src/relativetime/provider.rs

```diff
@@ -101,17 +104,20 @@ pub struct PluralCategoryStr<'data>(pub PluralCategory, pub Cow<'data, str>);
     databake(path = icu_experimental::relativetime::provider)
 )]
 #[yoke(prove_covariance_manually)]
-pub struct SinglePlaceholderPluralPattern<'data> {
+pub struct PluralPattern<'data, B: PatternBackend<Store = str>> {
     /// Optional patterns for categories other than PluralCategory::Other
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub specials: VarZeroVec<'data, PluralCategoryStrULE>,
     /// The pattern for PluralCategory::Other
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub other: Cow<'data, str>,
```

Comment:
Nit: The documentation should ideally say that the `str` is the serialized store string, not the human-readable pattern string